marketplace/deployer_util/provision.py

```diff
@@ -658,26 +667,32 @@ def provision_service_account(schema, prop, app_name, namespace,
   return sa_name, add_preprovisioned_labels(manifests, prop.name)
 
 
-def provision_storage_class(schema, prop, app_name, namespace):
-  if prop.storage_class.ssd:
-    sc_name = dns1123_name('{}-{}-{}'.format(namespace, app_name, prop.name))
-    manifests = [{
-        'apiVersion': 'storage.k8s.io/v1',
-        'kind': 'StorageClass',
-        'metadata': {
-            'name': sc_name,
-        },
-        # Some intelligence might go here to determine what
-        # provisioner and configuration to use here and below.
-        'provisioner': 'kubernetes.io/gce-pd',
-        'parameters': {
-            'type': 'pd-ssd',
-        }
-    }]
-    return sc_name, add_preprovisioned_labels(manifests, prop.name)
-  else:
-    raise Exception('Do not know how to provision for property {}'.format(
-        prop.name))
+def provision_storage_class(schema, prop, app_name, namespace, provisioner):
```

Comment:
Will there be an associated change to our UI provisioning flows?

Reply:
Yes, the UI is trying to provision a hardcoded gce-pd as well

Reply:
Makes sense. This approach seems fine for here but we probably need to make some schema changes to fully accommodate this.